os/customizing-sshd.md

```diff
@@ -23,11 +23,11 @@ storage:
           # Use most defaults for sshd configuration.
           UsePrivilegeSeparation sandbox
           Subsystem sftp internal-sftp
+          UseDNS no
 
           PermitRootLogin no
           AllowUsers core
-          PasswordAuthentication no
-          ChallengeResponseAuthentication no
```

Comment:
It looks okay.  I suppose anyone needing something else not in the whitelist (e.g. GSSAPI) probably won't be referencing this example anyway.